build/Dockerfile

```diff
@@ -9,8 +9,11 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-FROM quay.io/bitnami/golang:1.14 as builder
-RUN go env GOPROXY
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
+FROM registry.access.redhat.com/ubi8-minimal:8.4 as builder
```

Comment:
Why not use the same ubi image further down, rather than using `8.4` in the builder and `8.3` in the final image?

Reply:
should be the same, so updated both to 8.4-200.1622548483